content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,142 @@
+[concepts]:https://gateway-api.sigs.k8s.io/concepts/api-overview/
+[guides]:https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+[install-crds]:https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/#install-the-crds
+[issue]:https://github.com/kubernetes-sigs/gateway-api/issues/new/choose
+[disc]:https://github.com/kubernetes-sigs/gateway-api/discussions/new
+[community]:https://gateway-api.sigs.k8s.io/contributing/community/
+
+## Release Highlights
+
+### Graduation to Beta
+
+The `v0.5.0` release is particularly historic because it marks the growth in
+maturity to a `v1beta1` release for some of our key APIs:
+
+- `GatewayClass`
+- `Gateway`
+- `HTTPRoute`
+
+This achievement was marked by the completion of several graduation criteria:
+
+- over a dozen [implementations][impl]
+- the first release of our conformance test suite
+- most of the API surface is actively being used
+- approval from Kubernetes SIG Network API reviewers
+
+To see what's in store for future releases check out the [next steps
+section](/first-beta.md#next-steps).
+
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+
+### Release Channels
+
+This release introduces the `experimental` and `standard` [release channels][ch]
+which enable a better balance of maintaining stability while still enabling
+experimentation and iterative development.
+
+The `standard` release channel includes:
+
+- resources that have graduated to beta
+- fields that have graduated to standard (no longer considered experimental)
+
+The `experimental` release channel includes everything in the `standard` release
+channel, plus:
+
+- `alpha` API resources
+- un-graduated fields that are considered experimental
+
+Release channels are used internally to enable iterative development with
+quick turnaround, and externally to indicate feature stability to implementors
+and end-users.
+
+For this release we've added the following experimental features:
+
+- [Route select Gateway listeners by port number](https://github.com/kubernetes-sigs/gateway-api/pull/1002)
+- [URL rewrites and path redirects](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+
+[ch]:https://gateway-api.sigs.k8s.io/concepts/versioning/#release-channels-eg-experimental-standard
+
+## Other Improvements
+
+For an exhaustive list of changes included in the `v0.5.0` release, please see
+the [v0.5.0 CHANGELOG entry][changelog].
+
+[changelog]:https://github.com/kubernetes-sigs/gateway-api/blob/master/CHANGELOG.md#v050
+
+## Next Steps
+
+As we continue to mature the API for production use cases, the following is a
+(non-exhaustive) list of some of the highlights for upcoming releases:
+
+- [GRPCRoute][gep1016] for [gRPC][grpc] traffic routing
+- [route delegation][pr1085]
+- Layer 4 API maturity: `v1beta1` for `TCPRoute`, `UDPRoute` and `TLSRoute`
```

Comment:
I'd make TCPRoute, UDPRoute and TLSRoute be hyperlinks.

Reply:
```suggestion
- Layer 4 API maturity: `v1beta1` for [TCPRoute](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/tcproute_types.go), [UDPRoute](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/udproute_types.go) and [TLSRoute](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/tlsroute_types.go)
```